index.js

```diff
@@ -103,12 +104,20 @@ Hooks.on("updateScene", (scene, data) => {
 
 // Add Controls
 Hooks.on("getSceneControlButtons", (controls) => {
-    if (!game.user.isGM || !canvas.worldExplorer?.enabled) return;
+    if (!game.user.isGM) return;
+    if (!canvas.worldExplorer?.enabled) {
+        if (canvas.worldExplorer?.active) {
+            // World Explorer tools active, but not enabled for this scene, thus
+            // activate top (token) controls instead, so the scene doesn't fail to load
+            canvas.tokens.activate();
+        }
+        return;
+    }
 
     controls.worldExplorer = {
         name: "worldExplorer",
         title: game.i18n.localize("WorldExplorer.Name"),
-        icon: "fa-solid fa-map",
+        icon: "fa-solid fa-globe",
```

Comment:
The others might be ok, but I think to not break muscle memory it might be best to keep this one the same.